demos/04 Hurricane Ida Case Study/04 Hurricane Ida Offshore Rigs.ipynb

```diff
@@ -536,7 +536,6 @@
     "res_list = []\n",
     "for b in (\n",
     "    Blob.search()\n",
-    "    .filter(p.namespace == f\"{org}:{user_id}\")\n",
```

Comment:
presumably not needed because the async_func.id is unique to the user?

Reply:
didn't need it at all, I had honestly already removed this and I guess didn't push